source/_patterns/00-atoms/references/reference.mustache

```diff
@@ -1,5 +1,7 @@
 <div class="reference"{{#bibId}} data-popup-label="See in references" data-popup-contents="{{bibId}}" id="{{bibId}}"{{/bibId}}>
-
+  {{#referenceLabel}}
```

Comment:
Do you have any objection to just calling this `label` and dropping the prefix?

Reply:
Or `type`?

Reply:
I have replied to this in the main ticket. https://app.zenhub.com/workspaces/journal-5ede53e8b9b11773fd13f981/issues/elifesciences/issues/6468